parquet_file/tests/metadata.rs

```diff
@@ -81,6 +81,24 @@ async fn test_decoded_iox_metadata() {
         got, meta,
         "embedded metadata does not match original metadata"
     );
+
+    // Repro of 4695
+    let row_group_meta = decoded.parquet_row_group_metadata();
+    println!("ow_group_meta: {:#?}", row_group_meta);
```

Comment:
Typo, might cause debugging headaches if someone's doing ctrl-f for `row_group_meta`:

```suggestion
    println!("row_group_meta: {:#?}", row_group_meta);
```